docs/docs/cmd/pa/app/app-permission-list.mdx

```diff
@@ -0,0 +1,124 @@
+import Global from '/docs/cmd/_global.mdx';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# pa app permission list
+
+Lists all permissions of a Power Apps app
+
+## Usage
+
+```sh
+m365 pa app permission list [options]
+```
+
+## Options
+
+```md definition-list
+`---appName <appName>`
+: The name (GUID) of the Microsoft Power App.
+
+`--asAdmin`
+: Run the command as admin for apps you don't own.
+
+`-e, --environmentName [environmentName]`
+: The name of the environment. Required when using `asAdmin`.
+
+`--roleName [roleName]`
+: Filter the results to only a given role: `Owner`, `CanEdit`, `CanView`.
```

Comment:
These values might not be that clear for everyone. Let's add a remarks section where we elaborate on them.
You can take [flow list](https://pnp.github.io/cli-microsoft365/cmd/flow/flow-list#remarks) as an example.
Let's explain the values we use with the one shown in the UI (`owner`, `co-owner`, `user`).